data/STRchive-database.json

```diff
@@ -1207,7 +1207,7 @@
         "Mechanism_source":"(OMIM)",
         "source":"Seixas et al 2017 AJHG, NBK541729, s40478-021-01201-x",
         "notes":"Novel. Normal: [(ATTTT)7\u2013400] Pathogenic: [(ATTTT)60\u201379(ATTTC)31\u201375(ATTTT)58\u201390], ATTTC within (ATTTT)7\u2013400 repeat region",
-        "details":null,
+        "details":"Pathogenicity only associated with pathogenic motif >30 repeats, flanked by at least 58 repeats of reference motif on either side; normal repeat can range from 1 to 400 repeats, although typically less than 30 (NBK541729). The pathogenic motif is unstable, particularly when transmitted by the father (NBK541729).",
```

Comment:
normal repeat == reference motif? Maybe state the specific motif for clarity.

Reply:
fixed!